Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM python:3.7-slim-buster as ge-build
-RUN apt-get update
+RUN apt-get update \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get install -y --no-install-recommends
```

Comment:
ln 2 updates the list, and then ln 3 removes the lists .... and what is being installed on ln4?

since ge-build is a throwa way stage we dont have to clean up or optimize the final size